gatsby-theme-firebase/src/components/icons/Facebook.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+interface Props extends React.SVGAttributes<SVGElement> {
+    size?: number;
+}
+
+const Faceboook: React.FunctionComponent<Props> = ({
+    size = 16,
+    ...restProps
+}) => {
+    const aspectRatio = 98 / 120;
+    const height = size * aspectRatio;
+    return (
+        <svg width={size} height={height} viewBox="0 0 120 98" {...restProps}>
```

Comment:
Let's fix the svg size. We can remove `aspectRatio` and `height`.
And we can update the viewBox to `0 0 50 50`. 

```diff
-    const aspectRatio = 98 / 120;
-    const height = size * aspectRatio;
     return (
-        <svg width={size} height={height} viewBox="0 0 120 98" {...restProps}>
+        <svg width={size} height={size} viewBox="0 0 50 50" {...restProps}>
```